config/cscs.py

```diff
@@ -340,29 +340,34 @@ class ReframeSettings:
             'kesch': {
                 'PrgEnv-pgi-nompi': {
                     'type': 'ProgEnvironment',
-                    'modules': ['PrgEnv-pgi/17.10'],
+                    'modules': ['PE/17.06',
+                                'PrgEnv-pgi/18.05'],
                     'cc': 'pgcc',
                     'cxx': 'pgc++',
                     'ftn': 'pgf90',
                 },
                 'PrgEnv-pgi': {
                     'type': 'ProgEnvironment',
-                    'modules': ['PrgEnv-pgi/17.10_gdr'],
+                    'modules': ['PE/17.06',
+                                'PrgEnv-pgi/18.05'],
```

Comment:
What's the difference between this and the previous one?

Reply:
Now, there is only a single (with respect to mpi/nompi) PGI programming environment available on the system. So, the only difference remaining in this config file would be the compilers used, i. e. PGI vs. MPI. 
The question then is, whether we want to keep both or not?

Reply:
Ok, got it. Let's keep both then.